src/App.js

```diff
@@ -2,44 +2,44 @@ import React, { useEffect } from "react";
 import { Router, Route, Switch } from "react-router-dom";
 import AppRouter from "./router";
 import { useDispatch, useSelector } from "react-redux";
-import { createHashHistory } from 'history';
+import { createHashHistory } from "history";
 import { toast, ToastContainer } from "react-toastify";
 import "react-toastify/dist/ReactToastify.css";
-import api from "../src/apiServices"
+import api from "../src/apiServices";
+import Terms from "./Components/Terms&Conditions/Terms";
 import CoSetting from "./Components/OnBording/SubModules/DashBoardCO/components/CoSetting";
 function App() {
   return <MainApp />;
 }
 function MainApp() {
   // eslint-disable-next-line
   const state = useSelector((state) => state);
-  console.warn = (message, ...args) => { };
-  console.warn = () => { };
+  console.warn = (message, ...args) => {};
+  console.warn = () => {};
   const browserHistory = createHashHistory();
-  const userID = state && state.auth && state.auth.loginInfo && state.auth.loginInfo.UserID
-console.log(userID)
+  const userID =
+    state && state.auth && state.auth.loginInfo && state.auth.loginInfo.UserID;
+  console.log(userID);
```

Comment:
1) Removed all the console.logs.
2) use this state && state.auth?.loginInfo instead of state && state.auth && state.auth.loginInfo && state.auth.loginInfo.UserID.